spec/rubocop/cop/rspec/described_class_module_wrapping_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::DescribedClassModuleWrapping do
+  subject(:cop) { described_class.new }
+
+  context 'when an outermost describe block' do
+    it 'has no errors' do
```

Comment:
Personally, I don’t like using `context` blocks that do not set up a state (with `let` or `before`/`around`) and contain only one `it` block. I think it reads better to just have the `it` block in the parent scope. (This being rubocop-rspec, perhaps we should have a cop for checking this style 😄)

We tend to use wording like e.g. “allows”/“registers an offense” instead of “The “has no errors”/“has an error” in the spec examples. I went ahead and prepared four texts that you may want to use instead. I hope you don’t mind.

```
it 'allows a describe block in the outermost scope'
it 'registers an offense when RSpec.describe is nested within a module'
it 'registers an offense when RSpec.describe is nested within two modules'
it 'allows a module that does not contain RSpec.describe'
```

Reply:
Sweet. I'll get these integrated. Thanks for the review!